README.md

```diff
@@ -1 +1,69 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+Hello Microverse!
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/charpeamol)
+- Twitter: [@twitterhandle](https://twitter.com/charpe_amol)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/amol-charpe-762a69141/)
```

Comment:
- [ ] [_Optional_] Instead of `@githubhandle`, you can add your actual GitHub handle here like `@charpeamol ` so the user sees your handle in the ReadMe as a link.